packages/core/src/Modules/Cashier/Components/Error/deposit-locked.jsx

```diff
@@ -0,0 +1,127 @@
+    const has_poi_submitted = identity.status !== 'none';
+    const has_poa_submitted = document.status !== 'none';
+    const poi_text = has_poi_submitted
+        ? localize('Check proof of identity document verification status')
+        : localize('Upload an ID document to verify your identity');
+    const poa_text = has_poa_submitted
+        ? localize('Check proof of address document verification status')
+        : localize('Upload a proof of address to verify your address');
+    const history = useHistory();
+
+    // handle TnC
+    const acceptTnc = async () => {
+        await WS.tncApproval();
+        await WS.getSettings();
+        onMount();
+    };
+
+    // handle all deposits lock status
+    const items = [
+        ...(is_poi_needed
+            ? [
+                  {
+                      content: poi_text,
+                      status: 'action',
+                      onClick: () => history.push(routes.proof_of_identity),
+                  },
+              ]
+            : []),
+        ...(is_poa_needed
+            ? [
+                  {
+                      content: poa_text,
+                      status: 'action',
+                      onClick: () => history.push(routes.proof_of_address),
+                  },
+              ]
+            : []),
+        ...(is_tnc_needed
+            ? [
+                  {
+                      content: (
+                          <Localize
+                              i18n_default_text='Accept our updated <0>terms and conditions</0>'
+                              components={[
+                                  <a
+                                      key={0}
+                                      className='link'
+                                      rel='noopener noreferrer'
+                                      target='_blank'
+                                      href={getDerivComLink('terms-and-conditions')}
+                                  />,
+                              ]}
+                          />
+                      ),
+                      status: 'button-action',
+                      onClick: () => acceptTnc(),
+                      button_text: localize('I accept'),
+                  },
+              ]
+            : []),
+        ...(is_trading_experience_incomplete || (is_financial_account && is_financial_information_incomplete)
+            ? [
+                  {
+                      content: localize('Complete the financial assessment form'),
+                      status: 'action',
+                      onClick: () => history.push(routes.financial_assessment),
+                  },
+              ]
+            : []),
+    ];
+    return (
+        <div className='cashier-locked'>
+            <Icon icon='IcCashierNoBalance' className='cashier-locked__icon' />
+            <h2 className='cashier-locked__title'>{localize('Deposits are locked')}</h2>
+
+            {items.length ? (
+                <>
```

Comment:
isn't <> is an abbreviation of React.Fragment?

Reply:
@mustofa-binary Yes, but if you recall, according to the last discussion we had for code style, we decided to go with `React.Fragment` instead of `<>`.